libraries/report.rb

```diff
@@ -61,8 +55,6 @@ class ComplianceReport < Chef::Resource
           else
             Chef::Log.warn "#{collector} is not a supported inspec report collector"
           end
-
-          raise "#{total_failed} audits have failed.  Aborting chef-client run." if total_failed > 0 && node['audit']['fail_if_any_audits_failed']
```

Comment:
we no longer have a summary in the new format, which means we don't have failed counts, ya :/